docs/02-app/02-api-reference/05-next-config-js/expireTime.mdx

```diff
@@ -16,4 +16,6 @@ module.exports = {
 }
 ```
 
-Now instead of the default of 1 year `stale-while-revalidate` period being provided in the `Cache-Control` header, the custom period will be included.
+Now when sending the `Cache-Control` header the expire time will be calculated depending on the specific revalidate period.
+
+For example, if you have a revalidate of 30 seconds on a path and the expire time is 31536000 seconds the generated `Cache-Control` header will be `s-maxage=30, stale-while-revalidate=31535970` so that it can stay stale for 30 seconds less than the configured expire time.
```

Comment:
Having two numbers so far apart doesn't really show the difference since who cares?

Maybe an example of 15 minutes and expire in 60 minutes shows it better?